post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+:context: creating-multi-arch-compute-nodes-ibm-z
+[id="creating-multi-arch-compute-nodes-ibm-z"]
+= Creating a cluster with multi-architecture compute machine on {ibmzProductName} and {linuxoneProductName} with z/VM
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+To create a cluster with multi-architecture compute machines on {ibmzProductName} and {linuxoneProductName}(`s390x`) with z/VM, you must have an existing single-architecture bare metal (`x86_64`) cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user provisioned cluster on bare metal]. You can then add `s390x` compute machines to your {product-title} cluster on bare metal.
```

Comment:
you must have an existing single-architecture bare metal (x86_64) cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user provisioned cluster on bare metal]. You can then add s390x compute machines to your {product-title} cluster on bare metal.
-->
you must have an existing single-architecture x86_64 cluster.

(remove sentence "For more information ...")

Reply:
You can then add `s390x` compute machines to your {product-title} cluster on bare metal.
-->
You can then add `s390x` compute machines to your {product-title} cluster.